core/src/main/scala/cats/syntax/apply.scala

```diff
@@ -40,6 +41,205 @@ private[syntax] trait ApplySyntaxBinCompat0 {
     new IfApplyOps[F](fa)
 }
 
+private[syntax] trait ApplySyntaxBinCompat1 {
+  implicit final def catsSyntaxApplyFABOps[F[_], A, B](fab: F[A => B]): ApplyFABOps[F, A, B] =
+    new ApplyFABOps[F, A, B](fab)
+
+  implicit final def catsSyntaxApply2Ops[F[_], A, B, C](ff: F[(A, B) => C]): Apply2Ops[F, A, B, C] =
+    new Apply2Ops[F, A, B, C](ff)
+
+  implicit final def catsSyntaxApplyOps2[F[_], A](fa: F[A]): ApplyOps2[F, A] =
+    new ApplyOps2[F, A](fa)
+}
+
+final class ApplyFABOps[F[_], A, B](private val fab: F[A => B]) extends AnyVal {
+
+  /**
+   * @see [[Apply.ap]].
+   * 
+   * Example:
+   * {{{
+   * scala> import cats.syntax.all._
+   *
+   * scala> val someF: Option[Int => Long] = Some(_.toLong + 1L)
+   * scala> val noneF: Option[Int => Long] = None
+   * scala> val someInt: Option[Int] = Some(3)
+   * scala> val noneInt: Option[Int] = None
+   *
+   * scala> someF.ap(someInt)
+   * res0: Option[Long] = Some(4)
+   *
+   * scala> noneF.ap(someInt)
+   * res1: Option[Long] = None
+   *
+   * scala> someF.ap(noneInt)
+   * res2: Option[Long] = None
+   *
+   * scala> noneF.ap(noneInt)
+   * res3: Option[Long] = None
+   * }}}
+   */
+  def ap(fa: F[A])(implicit F: Apply[F]): F[B] = F.ap(fab)(fa)
+
+  /**
+   * Alias for [[ap]].
+   */
+  def <*>(fa: F[A])(implicit F: Apply[F]): F[B] = F.<*>[A, B](fab)(fa)
+}
+
+final class Apply2Ops[F[_], A, B, C](private val ff: F[(A, B) => C]) extends AnyVal {
```

Comment:
Maybe `ApplyFABCOps` for consistency with `ApplyFABOps`? We could also name them `ApplyFunctionOps` and `ApplyFunction2Ops` maybe.